backend/routes/api/seasons.js

```diff
@@ -0,0 +1,87 @@
+import { Router } from "express";
+import {
+  Park,
+  Season,
+  FeatureType,
+  Feature,
+  DateType,
+  DateRange,
+  Dateable,
+} from "../../models/index.js";
+import asyncHandler from "express-async-handler";
+
+const router = Router();
+
+router.get(
+  "/seasons/:seasonId",
+  asyncHandler(async (req, res) => {
+    const { seasonId } = req.params;
+
+    const seasonModel = await Season.findByPk(seasonId, {
+      attributes: ["id", "operatingYear", "status"],
+      include: [
+        {
+          model: FeatureType,
+          as: "featureType",
+          attributes: ["id", "name"],
+        },
+        {
+          model: Park,
+          as: "park",
+          attributes: ["id", "name", "orcs"],
+        },
+      ],
+    });
+
+    if (!seasonModel) {
+      res.status(404);
+      throw new Error("Season not found");
```

Comment:
I think throwing the error may cancel anything set on the `res` object up to that point.

We  have a cheap exception handler defined in the main index.js file right now that looks for a message and a status in the error object. I think you can do something like this 

```js
const error = new Error("Season not found");
error.status = 404;
throw error;
```

And then it will send the 404 status code to the browser when it shows the error page